docker-compose.yml

```diff
@@ -14,16 +14,10 @@ services:
       - .:/var/task
       - node_modules_folder:/var/task/node_modules
       - postgres_folder:/var/task/bin/postgres-11.4
-
-  builder:
-    <<: *app_base
-    image: pgdump-builder
-    build:
-      context: .
-      args:
-        - PG_VERSION=11.4
-      target: builder
-    command: bin/makezip.sh
+    ports:
+      - "8080:8080"
+    expose:
+      - 8080
```

Comment:
Since we changed to the AWS Lambda Images, the only way these images allow to test the handlers is via POST. Meaning you have to expose the internal RIC. Check usage on [https://gallery.ecr.aws/lambda/nodejs](https://gallery.ecr.aws/lambda/nodejs)